onnx/test/optimizer_test.py

```diff
@@ -122,6 +122,21 @@ def check_identity(node):  # type: (NodeProto) -> None
         assert len(optimized_model.graph.node[2].attribute[0].g.output) == 2
         assert optimized_model.graph.node[2].attribute[0].g.output[1].name == "_Y"
 
+    def test_eliminate_identity_graph_output(self):  # type: () -> None
+        add = helper.make_node("Add", ["X", "Y"], ["A"])
+        identity = helper.make_node("Identity", ["A"], ["B"])
+        graph = helper.make_graph(
+            [add, identity],
+            "test",
+            [helper.make_tensor_value_info("X", TensorProto.FLOAT, (5,)),
+             helper.make_tensor_value_info("Y", TensorProto.FLOAT, (5,))],
+            [helper.make_tensor_value_info("B", TensorProto.FLOAT, (5,))])
+        optimized_model = self._optimized(graph, ["eliminate_identity"])
+
+        for node in optimized_model.graph.node:
+            assert node.op_type != "Identity"
+        assert len(optimized_model.graph.node) == 1
```

Comment:
Would it be suitable to check if output shape is valid here?

Reply:
> Does the current master branch break this test?

The current master branch will fail on [this method](https://github.com/onnx/onnx/pull/2574/files#diff-0653f936835022eecf61d5127741c081R134), which checks the optimized model and find the output shape is missing.